src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java

```diff
@@ -0,0 +1,149 @@
+
+import com.ioet.bpm.organizations.boundaries.OrganizationController;
+import com.ioet.bpm.organizations.domain.Organization;
+import com.ioet.bpm.organizations.repositories.OrganizationRepository;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.internal.matchers.Or;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BpmOrganizationsApiApplicationTests {
+
+    @Mock
+    private OrganizationRepository organizationRepository;
+
+    @InjectMocks
+    private OrganizationController organizationController;
+
+
+    @Test
+    public void createOrganizationResponseSuccess() {
+        Organization organizationCreated = mock(Organization.class);
+        Organization organizationToCreate = mock(Organization.class);
+
+        ResponseEntity<Organization> organizationResponse = organizationController.createOrganization(organizationToCreate);
+
+        verify(organizationRepository).save(organizationToCreate);
+        assertEquals(HttpStatus.CREATED, organizationResponse.getStatusCode());
+
+    }
+
+
+    @Test
+    public void testGetAllOrganizationsUseTheRepository() {
+
+        ResponseEntity<Iterable> organizations = organizationController.getAllOrganizations();
+
+        assertEquals(HttpStatus.OK, organizations.getStatusCode());
+
+        verify(organizationRepository).findAll();
+    }
+
+    @Test
+    public void testGetOrganizationByIdReturnAOrganization() {
+        Organization organizationCreated = mock(Organization.class);
+        Organization organizationToCreate = new Organization();
+        organizationToCreate.setId("id");
+        organizationToCreate.setName("ioet");
+        when(organizationRepository.save(organizationToCreate)).thenReturn(organizationCreated);
+
+        ResponseEntity<Organization> organizationCreatedResponse = organizationController.createOrganization(organizationToCreate);
+
+        assertEquals(organizationCreated, organizationCreatedResponse.getBody());
+        assertEquals(HttpStatus.CREATED, organizationCreatedResponse.getStatusCode());
+        verify(organizationRepository).save(organizationToCreate);
+    }
+
+    @Test
+    public void whenAOrganizationIsFoundThenReturn200() {
+        Organization organizationFound = mock(Organization.class);
+        String organizationFoundId = "id";
+        when(organizationRepository.findById(organizationFoundId)).thenReturn(Optional.of(organizationFound));
+
+        ResponseEntity response = organizationController.getOrganization(organizationFoundId);
+
+
+        verify(organizationRepository).findById(organizationFoundId);
```

Comment:
We don't need many empty lines here, one is enough.

Reply:
Done